---
layout: news
title: Publications
subtitle: Publications in Wang Lab
permalink: /publications
---

# Publications

Please visit [Google Scholar](https://scholar.google.com/citations?user=ZwKmcpYAAAAJ&hl=en) for more details.<br>
\# Co-first authors; \* Co-corresponding authors

### 2025
- **Yan Y#**, **Sun D#**, **Hu J#**, **Sun L**, Yu H, Xiong Y, Huang Z, Xia H, Zhu X, Bian D, Sun F, Chen Y, Hou L, Wu C, Fan R, Zeng A\*, Zhang L\*, Sun Y\*, **Wang C\***, Zhang P\*. Multi-omic profiling highlights factors associated with resistance to immuno-chemotherapy in non-small cell lung cancer. _**Nat. Genet.**_ 2025; 57 (1), 126-139.<br>
[[Abstract]](https://www.nature.com/articles/s41588-024-01998-y)
[[Full Paper]](/resources/publications/2024_NG_NSCLC.pdf)

### 2024
- Liu S#\*, Feng C#, Tan L#, Zhang D, Li Y, **Han Y\***, **Wang C\***. Single-cell dissection of multifocal bladder cancer reveals malignant and immune cells variation between primary and recurrent tumor lesions. _**Commu. Biol.**_ 2024; 7 (1), 1659.<br>
[[Abstract]](https://www.nature.com/articles/s42003-024-07343-7?utm_source=rct_congratemailt&utm_medium=email&utm_campaign=oa_20241219&utm_content=10.1038/s42003-024-07343-7)
[[Full Paper]](/resources/publications/2024_CB_BLCA.pdf)
- **Chang Z#**, **Xu Y#**, **Dong X**, Gao Y, **Wang C\***. Single-cell and spatial multiomic inference of gene regulatory networks using SCRIPro. _**Bioinform.**_ 2024; 40 (7), btae466.<br>
[[Abstract]](https://academic.oup.com/bioinformatics/advance-article-abstract/doi/10.1093/bioinformatics/btae466/7716542?utm_source=advanceaccess&utm_campaign=bioinformatics&utm_medium=email)
[[Full Paper]](/resources/publications/2024_Bioinform_SCRIPro.pdf)
[[Software]](https://github.com/wanglabtongji/SCRIPro)
- Sun F#, Li H#, **Sun D#**, Fu S#, Gu L#, Shao X#, Wang Q#, **Dong X#**, Duan B#, Xing F#, Wu J#, Xiao M\*, Zhao F\*, Han J\*, Liu Q\*, Fan X\*, Li C\*, **Wang C\***, Shi T\*. Single-Cell Omics: experimental workflow, data analyses and applications. _**Sci. China Life Sci.**_ 2024; 1-98. <br>
[[Abstract]](https://www.sciengine.com/SCLS/doi/10.1007/s11427-023-2561-0;JSESSIONID=7b2d6e31-7cb2-46e4-ac9e-d8006970d173)
[[Full Paper]](/resources/publications/2024_SCLS_SCReview.pdf)
- Liu Q#, Zhang J#, Guo C#, Wang M#, **Wang C#**, Yan Y, Sun L, Wang D, Zhang L, Yu H, Hou L, Wu C, Zhu Y, Jiang G, Zhu H, Zhou Y, Fang S, Zhang T, Hu L, Li J, Liu Y, Zhang H, Zhang B, Ding L, Robles A, Rodriguez H, Gao D\*, Ji H\*, Zhou H\*, Zhang P\*. Proteogenomic characterization of small cell lung cancer identifies biological insights and subtype-specific therapeutic strategies. _**Cell**_ 2024; 187 (1), 184-203.<br>
[[Abstract]](https://www.cell.com/cell/pdf/S0092-8674(23)01335-1.pdf)
[[Full Paper]](/resources/publications/2024_Cell_SCLC.pdf)

### 2023
- **Ren P#**, **Shi X#**, **Dong X**, Yu Z, Ding X, Wang J, Sun L, Yan Y, Hu J, Zhang P, Chen Q, Li T, **Wang C\***. SELINA: Single-cell Assignment using Multiple-Adversarial Domain Adaptation Network with Large-scale References. _**Cell Rep. Methods**_ 2023; 100577.<br>
[[Abstract]](https://www.cell.com/cell-reports-methods/fulltext/S2667-2375(23)00221-7)
[[Full Paper]](/resources/publications/2023_CRM_SELINA.pdf)
[[Software]](https://github.com/wanglabtongji/SELINA.py)
- **Wei H#**, **Han T**, Li T, **Wu Q\***, **Wang C\***. SCREE: a comprehensive pipeline for single-cell multi-modal CRISPR screen data processing and analysis. _**Brief. Bioinformatics**_ 2023; 24(3),bbad123.<br>
[[Abstract]](https://academic.oup.com/bib/advance-article-abstract/doi/10.1093/bib/bbad123/7095415?utm_source=advanceaccess&utm_campaign=bib&utm_medium=email)
[[Full Paper]](/resources/publications/2023_BIB_SCREE.pdf)
[[Software]](https://github.com/wanglabtongji/SCREE)
- **Han Y#**, **Wang Y#**, **Dong X#**, **Sun D**, **Liu Z**, **Yue J**, Wang H, Li T\*, **Wang C\***. TISCH2: expanded datasets and new tools for single-cell transcriptome analyses of the tumor microenvironment. _**Nucleic Acids Res.**_ 2023; 51(D1),D1425-D1431.<br> 
[[Abstract]](https://academic.oup.com/nar/advance-article/doi/10.1093/nar/gkac959/6793806?login=false)
[[Full Paper]](/resources/publications/2022_NAR_TISCH.pdf)
[[Software]](http://tisch.comp-genomics.org/)
- **Shi X#**, Yu Z#, **Ren P**, **Dong X**, Ding X, Song J, Zhang J, Li T\*, **Wang C\***. HUSCH: an integrated single-cell transcriptome atlas for human tissue gene expression visualization and analyses. _**Nucleic Acids Res.**_ 2023; 51(D1),D1029-D1037.<br>
[[Abstract]](https://academic.oup.com/nar/advance-article/doi/10.1093/nar/gkac1001/6786201?login=false)
[[Full Paper]](/resources/publications/2022_NAR_HUSCH.pdf)
[[Software]](http://husch.comp-genomics.org/)
- **Han T#**, Wang X#, Shi S, Zhang W, Wang J, **Wu Q**, Li Z, Fu J, Zheng R, Zhang J, Tang Q, Zhang P\*, **Wang C\***. Cancer Cell Resistance to IFNγ Can Occur via Enhanced Double-Strand Break Repair Pathway Activity. _**Cancer Immunol Res.**_ 2023; 11(3),381–398.<br>
[[Abstract]](https://aacrjournals.org/cancerimmunolres/article-abstract/doi/10.1158/2326-6066.CIR-22-0056/716422/Cancer-Cell-Resistance-to-IFN-Can-Occur-via)
[[Full Paper]](/resources/publications/2023_CIR_INFG.pdf)
- **Hu J#**, Zhang L#, Xia H#, **Yan Y#**, Zhu X, Sun F, Sun L, Li S, Li D, **Wang J**, **Han Y**, Zhang J, Bian D, Yu H, Chen Y, Fan P, Ma Q, Jiang G, **Wang C\***, Zhang P\*. Tumor microenvironment remodeling after neoadjuvant immunotherapy in non-small cell lung cancer revealed by single-cell RNA sequencing. _**Genome Med.**_ 2023; 15(1),1-14.<br>
[[Abstract]](https://link.springer.com/article/10.1186/s13073-023-01164-9)
[[Full Paper]](/resources/publications/2023_GM_NSCLC.pdf)
- Cao G#, **Yue J#**, Ruan Y#, **Han Y**, Zhi Y, Lu J, Liu M, Xu X, Wang J, Gu Q, Wen X, Gao J, Kang J, Zhang Q, **Wang C\***, Li F\*. Single-cell Dissection of Cervical Cancer Reveals Key Subsets of the Tumor Immune Microenvironment. _**EMBO J.**_ 2023; 42,e110757. <br>
[[Abstract]](https://www.embopress.org/doi/abs/10.15252/embj.2022110757)
[[Full Paper]](/resources/publications/2023_EMBOJ_CESC.pdf)
- Sahu A#\*, Wang X#, Munson P#, Klomp J, Wang X, Gu S, **Han Y**, Qian G, Nicol P, Zeng Z, **Wang C**, Tokheim C, Zhang W, Fu J, Wang J, Nair N, Rens J, Bourajjaj M, Jansen B, Leenders I, Lemmers J, Musters M, Zanten S, Zelst L, Worthington J, Liu J, Juric D, Meyer C, Oubrie A, Liu X, Fisher D\*, Flaherty K\*. Discovery of Targets for Immune–Metabolic Antitumor Drugs Identifies Estrogen-Related Receptor Alpha. _**Cancer Discov.**_ 2023; 13(3),672-701.<br>
- Zhang Y, Xiang G, Jiang AY, Lynch A, Zeng Z, **Wang C**, Zhang W, Fan J, Kang J, Gu S, Wan C, Zhang B, Liu XS\*, Brown M\*, Meyer CA\*. MetaTiME integrates single-cell gene expression to characterize the meta-components of the tumor immune microenvironment. _**Nat. Commun.**_ 2023; 14(1),2634.<br>

### 2022
- **Sun D#**, **Liu Z**, Li T, **Wu Q\***, **Wang C\***. STRIDE: accurately decomposing and integrating spatial transcriptomics using single-cell RNA sequencing. _**Nucleic Acids Res.**_ 2022; 50(7),e42-e42.<br>
[[Abstract]](https://academic.oup.com/nar/article-abstract/50/7/e42/6543547)
[[Full Paper]](/resources/publications/2022_NAR_STRIDE.pdf)
[[Software]](https://github.com/wanglabtongji/STRIDE)
- **Dong X#**, **Tang K#**, **Xu Y**, **Wei H**, **Han T**, **Wang C\***. Single-cell Gene Regulation Network Inference by Large-scale Data Integration. _**Nucleic Acids Res.**_ 2022; 50(21),e126-e126.<br>
[[Abstract]](https://academic.oup.com/nar/advance-article/doi/10.1093/nar/gkac819/6717821?login=false)
[[Full Paper]](/resources/publications/2022_NAR_SCRIP.pdf)
[[Software]](https://github.com/wanglabtongji/SRIP)
- **Liu Z#**, **Sun D**, **Wang C\***. Evaluation of cell-cell interaction methods by integrating single-cell RNA sequencing data with spatial information. _**Genome Biol.**_ 2022; 23(1),1-38.<br> 
[[Abstract]](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-022-02783-y)
[[Full Paper]](/resources/publications/2022_GB_CCI.pdf)
[[Software]](https://github.com/wanglabtongji/CCI)
- Xu R#, Li S#, **Wu Q#**, Li C#, Jiang M#, Guo L, Chen M, Yang L, **Dong X**, Wang H, **Wang C\***, Liu X\*, Ou X\*, Gao S\*. Stage-specific H3K9me3 occupancy ensures retrotransposon silencing in human preimplantation embryos. _**Cell Stem Cell**_ 2022; 29(7),1051-1066. **Cover Story**<br>
[[Abstract]](https://www.sciencedirect.com/science/article/abs/pii/S1934590922002508)
[[Full Paper]](/resources/publications/2022_CSC_H3K9me3.pdf)
- **Wang C#**, Chen C#, Liu X#, Li C#, Wu Q, Chen X, Yang L, Kou X, Zhao Y, Wang H, Gao Y\*, Zhang Y\*, Gao S\*. Dynamic nucleosome organization after fertilization reveals regulatory factors for mouse zygotic genome activation. _**Cell Res.**_ 2022; 32(9),801-813. **Cover Story**<br>
[[Abstract]](https://www.nature.com/articles/s41422-022-00652-8)
[[Full Paper]](/resources/publications/2022_CR_Nucleosome.pdf)
- Yang H#, Bai D#, Li Y#, Yu Z#, **Wang C**, Sheng Y, Liu W\*, Gao S\*, Zhang Y\*. Allele-specific H3K9me3 and DNA methylation co-marked CpG-rich regions serve as potential imprinting control regions in pre-implantation embryo. _**Nat. Cell Biol.**_ 2022;  24(5),783-792.<br>
- Zheng W#\*, Wang X#\*, Liu J, Yu X, Li L, Wang H, Yu J, Pei X, Li C, Wang Z, Zhang M, Zeng X, Zhang F, Wang C, Chen H, Chen H\*. Single-cell analyses highlight the proinflammatory contribution of C1q-high monocytes to Behçet's disease. _**PNAS**_ 2022; 19(26),e2204289119.<br>

### 2021
- **Sun D#**, **Wang J#**, **Han Y#**, **Dong X**, **Zheng R**, **Ge J**, **Shi X**, **Wang B**, **Ren P**, Sun L, Yan Y, Zhang P, Zhang F\*, Li T\*, **Wang C\***. TISCH: a comprehensive web resource enabling interactive single-cell transcriptome visualization of tumor microenvironment. _**Nucleic Acids Res.**_ 2021; 49(D1),D1420-D1430.<br>
[[Abstract]](https://pubmed.ncbi.nlm.nih.gov/33179754/)
[[Full Paper]](/resources/publications/2020_NAR_TISCH.pdf)
[[Website]](http://tisch.comp-genomics.org/)
- Zhang H#, Song L#, Wang X, Cheng H, **Wang C**, Meyer CA, Liu T, Tang M, Aluru S, Yue F, Liu XS\*, Li H\*. Fast alignment and preprocessing of chromatin profiles with Chromap. _**Nat. Commun.**_ 2021; 12(1),6566.<br>	
- Wu SZ#, Al-Eryani G#,  Roden DL#, Junankar S, Harvey K, Andersson A, Thennavan A, **Wang C**, Torpy JR, Bartonicek N, Wang T, Larsson L, Kaczorowski D, Weisenfeld NI, Uytingco CR, Chew JG, Bent ZW, Chan C, Gnanasambandapillai V, Dutertre CA, Gluch L, Hui M, Beith J, Parker A, Robbins E, Segara D, Cooper C, Mak C, Chan B, Warrier S, Ginhoux F, Millar E, Powell JE, Williams TR,Liu XS, O'Toole S, Lim E, Lundeberg J, Perou CM, Swarbrick A\*. A single-cell and spatially resolved atlas of human breast cancers. _**Nat. Genet.**_ 2021; 53(9),1334-1347.<br>

### 2020
- **Wang C#**, **Sun D#**, Huang X, Wan C, Li Z, **Han Y**, Qin Q, Fan J, Qiu X, Xie Y, Meyer CA, Brown M, Tang M, Long H, Liu T\*, Liu XS\*. Integrative analyses of single-cell transcriptome and regulome using MAESTRO. _**Genome Biol.**_ 2020; 21(1),1-28.<br>
[[Abstract]](https://pubmed.ncbi.nlm.nih.gov/32767996)
[[Full Paper]](/resources/publications/2020_GB_MAESTRO.pdf)
[[Software]](http://github.com/liulab-dfci/MAESTRO)
- Burton A, Brochard V, Galan C, Ruiz-Morales E, Rovira Q, Rodriguez-Terrones D, Kruse K, Gras S, Udayakumar V, Chin H, Eid A, Liu X, **Wang C**, Gao S, Pradhan S, Vaquerizas J, Beaujean N, Jenuwein T, Torres-Padilla M. Heterochromatin establishment during early mammalian development is regulated by pericentromeric RNA and characterized by non-repressive H3K9me3. _**Nat. Cell Biol.**_ 2020; 22(7),767-778.<br>
- Wu SZ#, Roden D#, **Wang C**, Holliday H, Harvery K, Cazet AS, Murphy KJ, Pereira B, Al-Eryani G, Bartonicek N, Hou R, Torpy JR, Junankar S, Chan C, Lam CE, Hui M, Gluch L, Beith J, Parker A, Robbins E, Segara D, Mark C, Cooper C, Warrier S, Forrest A, Powell J, O’Toole S, Cox TR, Timpson P, Lim E, Liu XS, Swarbrick A\*. Stromal cell diversity associated with immune evasion in human triple-negative breast cancer. _**EMBO J.**_ 2020; 39(19),e104063.<br>
- Gu S#, Wang X#, Hu X#, Jiang P, Li Z, Traugh N, Bu X, Tang Q, **Wang C**, Zeng Z, Fu J, Meyer C, Zhang Y, Cejas P, Lim K, Wang J, Zhang W, Tokheim C, Sahu AD, Xing X, Kroger B, Ouyang Z, Long H, Freeman GJ\*, Brown M\*, Liu XS\*. Clonal tracing reveals diverse patterns of response to immune checkpoint blockade. _**Genome Biol.**_ 2020; 21,263.<br>

### Before 2020
- **Wang C#**, Liu X#, Gao Y#\*, Yang L#, Li C, Liu W, Chen C, Kou X, Zhao Y, Chen J, Wang Y, Le R, Wang H, Duan T, Zhang Y\*, Gao S\*. Reprogramming of H3K9me3-dependent heterochromatin during mammalian embryo development. _**Nat. Cell Biol.**_ 2018; 20(5),620-631.<br>
[[Abstract]](https://pubmed.ncbi.nlm.nih.gov/29686265)
[[Full Paper]](/resources/publications/2018_NCB_H3K9me3.pdf)
- Gao R#, **Wang C#**, Gao Y#, Bai D, Liu X, Kou X, Zhao Y, Zang R, Liao Y, Jia Y, Chen J, Wang H, Wan X, Liu W\*, Zhang Y\*, Gao S$\*. Inhibition of aberrant DNA re-methylation improves the development of nuclear transfer embryos. _**Cell Stem Cell**_ 2018; 23(3),426-435.<br>
[[Abstract]](https://pubmed.ncbi.nlm.nih.gov/30146410)
[[Full Paper]](/resources/publications/2018_CSC_NTmethyl.pdf)
- Liu X#, **Wang C#**, Liu W#, Li J#, Li C, Kou X, Chen J, Zhao Y, Gao H, Wang H, Zhang Y\*, Gao Y\*, Gao S\*. Distinct features of H3K4me3 and H3K27me3 chromatin domains in pre-implantation embryos. _**Nature**_ 2016; 537(7621),558-562. <br>
[[Abstract]](https://pubmed.ncbi.nlm.nih.gov/27626379)
[[Full Paper]](/resources/publications/2016_Nature_H3K4me3H3K27me3.pdf)
- Liu W#, Liu X#, **Wang C#**, Gao Y#, Gao R, Kou X, Zhao Y, Li J, Wu Y, Xiu W, Wang S, Yin J, Liu W, Cai T, Wang H, Zhang Y\*, Gao S\*. Identification of key factors conquering developmental arrest of somatic cell cloned embryos by combining embryo biopsy and single-cell sequencing. _**Cell Discov.**_ 2016; 2(1),1-15.<br>
[[Abstract]](https://pubmed.ncbi.nlm.nih.gov/27462457)
[[Full Paper]](/resources/publications/2016_CD_NTexpr.pdf)
- Gao R#, Xiu W#, Zhang L, Zang R, Yang L, **Wang C**, Wang M, Wang M, Yi L, Tang Y, Gao Y, Wang H, Xi J, Liu W, Wang Y, Wen X, Yu Y, Zhang Y, Chen L, Chen J\*, Gao S\*. Direct induction of neural progenitor cells transiently passes through a partially reprogrammed state. _**Biomaterials**_ 2017; 119,53-67.<br>
- Xu K#, Chen X#, Yang H, Xu Y, He Y, **Wang C**, Huang H, Liu B, Liu W, Li J, Kou X, Zhao Y, Zhao K, Zhang L, Hou Z, Wang H, Wang H, Li J, Fan H, Wang F, Gao Y, Zhang Y, Chen J\*, Gao S\*. Maternal Sall4 is indispensable for epigenetic maturation of mouse oocytes. _**J. Biol. Chem.**_ 2017; 292,1798-1807.<br>
- Wang Y, Wu Q, Yang P, **Wang C**, Liu J, Ding W, Liu W, Bai Y, Yang Y, Wang H, Gao S\*, Wang X\*. LSD1 co-repressor Rcor2 orchestrates neurogenesis in the developing mouse brain. _**Nat. Commun.**_ 2016; 7,10481.<br>
- Xu H#, Xu K#, He HH, Zang C, Chen CH, Chen Y, Qin Q, Wang S, **Wang C**, Hu S, Li F, Long H, Brown M\*, Liu XS\*. Integrative analysis reveals the transcriptional collaboration between EZH2 and E2F1 in the regulation of cancer-related gene expression. _**Mol. Cancer Res.**_ 2016; 14(2),163-72.<br>
- Zhang Y#\*, Vastenhouw NL#\*, Feng J, Fu K, **Wang C**, Ge Y, Pauli A, van Hummelen P, Schier AF\*, Liu XS\*. Canonical nucleosome organization at promoters forms during genome activation. _**Genome Res.**_ 2014;  24(2):260-6.<br>
- Wang J, Lin X, Wang S, **Wang C**, Wang Q, Duan X, Lu P, Wang Q, Wang C, Liu XS, Huang J\*. PHF8 and REST/NRSF co-occupy gene promoters to regulate proximal gene expression. _**Sci. Rep.**_ 2014; 4,5508.<br>
- Wang S, Sun Hanfei, Ma J, Zang C, **Wang C**, Wang J, Tang Q, Meyer CA, Zhang Y\*, Liu XS\*. Target analysis by integration of transcriptome and ChIP-seq data with BETA. _**Nat. Protoc.**_ 2013; 8,2502-15.<br>


